osquery/database/plugins/ephemeral.cpp

```diff
@@ -76,6 +80,22 @@ Status EphemeralDatabasePlugin::remove(const std::string& domain,
   return Status(0);
 }
 
+Status EphemeralDatabasePlugin::removeRange(const std::string& domain,
+                                            const std::string& low,
+                                            const std::string& high) {
+  std::vector<std::string> keys;
+  for (const auto& it : db_[domain]) {
```

Comment:
Does `DBType db_` here have any issue if, in the off chance, `domain` gets set to `""` above?